addons/im_livechat/static/tests/tours/im_livechat_channel_creation_tour.js

```diff
@@ -28,5 +29,21 @@ registry.category("web_tour.tours").add("im_livechat_request_chat_and_send_messa
         {
             trigger: ".o-livechat-root:shadow .o-mail-Message:contains('Hello, I need help')",
         },
+        {
+            trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+            run: "edit /",
+        },
+        {
+            trigger: "body",
+            async run() {
+                await contains(".o-mail-Composer-input", {
+                    parent: [".o-livechat-root", { shadowRoot: true }],
+                });
+                await contains(".o-mail-NavigableList-item", {
+                    count: 0,
+                    parent: [".o-livechat-root", { shadowRoot: true }],
+                });
+            },
+        },
```

Comment:
This check is not robust (as is often the case with negative checks): we have no guarantee that the `NavigableList` won't open after the contains call. Additionally, we cannot ensure that the `o-mail-NavigableList-item` corresponds to the actual list.